assets/js/aplus.js

```diff
@@ -118,6 +119,31 @@ $(function() {
     $(window).on('scroll', modifyMenu);
     $(window).on('resize', updateMenu);
 
+    var setSidebarState = function(collapsed) {
```

Comment:
> I don't know any reason why var x = function(){} has been used instead of function x() {}

Me neither. I changed all of my function variables into standard functions.